dotnet-roslyn/analyzers/Audacia.CodeAnalysis.Analyzers/src/Audacia.CodeAnalysis.Analyzers/Common/DiagnosticId.cs

```diff
@@ -17,5 +17,7 @@ public static class DiagnosticId
         public const string MaximumWhereClauses = "ACL1012";
         public const string UseRecordTypes = "ACL1013";
         public const string DoNotUseNumberInIdentifierName = "ACL1014";
+        public const string DoNotUseProducesResponseTypeWithTypedResults = "ACL1015";
```

Comment:
Please can you update [HelpLinkUrlFactory](https://github.com/audaciaconsulting/Audacia.CodeAnalysis/blob/master/dotnet-roslyn/analyzers/Audacia.CodeAnalysis.Analyzers/src/Audacia.CodeAnalysis.Analyzers/Common/HelpLinkUrlFactory.cs) to include these new ones?

Reply:
will do.